portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -0,0 +1,50 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import java.util.ArrayList;
+import java.util.List;
+import com.google.gson.Gson;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+@WebServlet("/comment")
+public class CommentServlet extends HttpServlet {
+
+  private final List<String> comments = new ArrayList<>();
+
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("application/json");
+    String json = new Gson().toJson(comments);
+    response.getWriter().println(json);
+  }
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String commenttext = getComment(request);
+    comments.add(commenttext);
+    response.sendRedirect("/index.html");
+  }
+  private String getComment(HttpServletRequest request) {
+    String comment = request.getParameter("comment");
```

Comment:
What happens if this get fails?

Reply:
it can't ever fail though since a string can be 0 characters or a lot of characters for that matter

Reply:
But, how would I go about solving this problem, should I use a try and catch approach?

Reply:
That assumes nothing goes wrong with the application. Take a look at the the oracle docs for this method, and see if you can come up with a few ways this can crash: https://docs.oracle.com/javaee/6/api/javax/servlet/ServletRequest.html#getParameter(java.lang.String). We can chat about it if you want.

Reply:
I fixed the try and catch method in pull request #11